physics/cires_ugwp_triggers_v1.F90

```diff
@@ -0,0 +1,576 @@
+module cires_ugwp_triggers_v1
+
+
+contains
+
+
+        subroutine ugwp_triggers
+        implicit none
+        write(6,*) ' physics-based triggers for UGWP ' 
+        end subroutine ugwp_triggers
+!	
+     SUBROUTINE  subs_diag_geo(nx, ny,  lat, lon, rlat, rlon, dy, dx,  &
+                               cosv, rlatc, brcos, brcos2, dlam1, dlam2, dlat, divJp, divJm)
+      use ugwp_common_v1 , only : deg_to_rad
```

Comment:
Constants should come from the host model through the argument list, if possible.

Reply:
Done.